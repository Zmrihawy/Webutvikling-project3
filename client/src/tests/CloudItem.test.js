import React from "react";
import renderer from "react-test-renderer";
import CloudItem from "../pages/dataVisualization/CloudItem";

it("renders cloud item", () => {
    const props = null;
    const tree = renderer.create(<CloudItem props={props} />).toJSON();
    expect(tree).toMatchSnapshot();
});
