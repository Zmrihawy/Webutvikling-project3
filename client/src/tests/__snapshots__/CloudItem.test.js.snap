// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders cloud item 1`] = `
<div
  className="tag-item-wrapper"
  props={null}
>
  <div />
  <div
    className="tag-item-tooltip"
  >
    HOVERED!
  </div>
</div>
`;
